setup.py

```diff
@@ -153,7 +153,8 @@ def run(self):
             "pandas-profiling>=3.1",
             "ipython>=7.0",
             "markdown>=3.3",
-            "Jinja2>=3.0",
+            "Jinja2>=2.11; platform_system != 'Windows'",
```

Comment:
@harupy To confirm, there weren't any code changes we needed to make for Unix Jinja 2.11 compatibility?

Reply:
No :)